eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/service/config/remote/RemoteRuntimeService.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.dashboard.console.service.config.remote;
+
+
+import org.apache.eventmesh.dashboard.console.entity.config.ConfigEntity;
+import org.apache.eventmesh.dashboard.console.entity.runtime.RuntimeEntity;
+import org.apache.eventmesh.dashboard.console.mapper.config.ConfigMapper;
+
+import java.util.ArrayList;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.alibaba.fastjson.JSON;
+import com.alibaba.fastjson.JSONObject;
+
+@Service
+public class RemoteRuntimeService {
+
+    @Autowired
+    private ConfigMapper configMapper;
+
+    public void getRuntimeConfigByCluster(RuntimeEntity runtimeEntity) {
+        RestTemplate restTemplate = new RestTemplate();
+        String forEntity =
+            restTemplate.getForObject("http://" + runtimeEntity.getHost() + ":" + runtimeEntity.getPort() + "/v2/configuration?format=properties", String.class, (Object) null);
+        JSONObject jsonMap = JSON.parseObject(forEntity);
+        Set<Entry<String, Object>> entries = jsonMap.entrySet();
+        ArrayList<ConfigEntity> configEntities = new ArrayList<>();
```

Comment:
RPC implements to EventMesh Runtime, Event Store and Meta should be put in the `core` module. The `console` module can call `org.apache.eventmesh.dashboard.service.remoting.ConfigRemotingService`.

This RemotingService interface needs to differentiate between methods based on the type of operation and the type of configuration, and abstract classes based on the target of the operation. I'll submit a PR for that.